Formula/voldemort.rb

```diff
@@ -4,7 +4,7 @@ class Voldemort < Formula
   url "https://github.com/voldemort/voldemort/archive/release-1.10.26-cutoff.tar.gz"
   sha256 "8bd41b53c3b903615d281e7277d5a9225075c3d00ea56c6e44d73f6327c73d55"
   license "Apache-2.0"
-  revision 1
+  revision 2
```

Comment:
Is the revision bump needed, or is setting `JAVA_HOME` in the test enough?

Reply:
It's in the install so this is needed

Reply:
Ah, misread. Thought it was the test block